neurom/core/types.py

```diff
@@ -28,11 +28,18 @@
 
 '''Type enumerations'''
 
-from enum import unique
+from enum import unique, Enum
 
 from neurom.utils import OrderedEnum
 
 
+@unique
+class NeuriteIter(Enum):
+    '''Neurite iteration orders'''
+    Natural = 1  # Order in which neurites appear in the file
+    NRN = 2  # NRN simulator order: soma -> axon -> basal -> apical
```

Comment:
is this internal to bbp NEURON, or generic for stock NEURON?

Reply:
@wvangeit do you know ?

Reply:
This is the order of stock NEURON:
https://github.com/neuronsimulator/nrn/blob/2dbf2ebf95f1f8e5a9f0565272c18b1c87b2e54c/share/lib/hoc/import3d/import3d_gui.hoc#L874

For neurodamus we're using stock NEURON import3d. (I know that the asc -> h5 converter is still out of date regarding that, @mgeplf can probably check if there relevant code is the same).

Reply:
cool, let's add that link to the comment, so people know where to look it up

Reply:
Thanks for the info Werner.